src/pages/billing/List.js

```diff
@@ -232,25 +232,20 @@ export const BillingList = _.flow(
           }
         }, [
           div(breadcrumbs),
-          div({
-            style: {
-              textOverflow: 'ellipsis', whiteSpace: 'nowrap',
-              fontSize: '1.25rem', overflowX: 'hidden'
-            }
-          }, [selectedName])
+          div({ style: Style.breadcrumb.textUnderBreadcrumb }, [selectedName])
```

Comment:
Not a strict refactor here because this adds a color style. Is this intended?

Reply:
Yup!